packages/create/package.json

```diff
@@ -40,6 +40,7 @@
     "dedent": "^0.7.0",
     "deepmerge": "^4.2.2",
     "glob": "^7.1.3",
+    "mocha": "^8.1.3",
```

Comment:
why add it?

Reply:
should be a dev dependency

Reply:
Yes, it was a mistake to be in dependencies. I have added it because when I tried to run `npm run test` it failed saying that could not found mocha. 

I will move it to `devDependencies`